addons/binding/org.openhab.binding.pioneeravr/src/main/java/org/openhab/binding/pioneeravr/internal/discovery/PioneerAvrDiscoveryParticipant.java

```diff
@@ -118,18 +126,51 @@ public ThingUID getThingUID(RemoteDevice device) {
 
     /**
      * Return true only if the given device model is supported.
-     * 
+     *
      * @param deviceModel
      * @return
      */
     private boolean isSupportedDeviceModel(final String deviceModel) {
         return StringUtils.isNotBlank(deviceModel)
                 && !Collections2.filter(PioneerAvrBindingConstants.SUPPORTED_DEVICE_MODELS,
                         new com.google.common.base.Predicate<String>() {
+                            @Override
                             public boolean apply(String input) {
                                 return StringUtils.startsWithIgnoreCase(deviceModel, input);
                             }
                         }).isEmpty();
     }
 
+    private boolean isSupportedDeviceModel2016(final String deviceModel) {
+        return StringUtils.isNotBlank(deviceModel)
+                && !Collections2.filter(PioneerAvrBindingConstants.SUPPORTED_DEVICE_MODELS2016,
+                        new com.google.common.base.Predicate<String>() {
+                            @Override
+                            public boolean apply(String input) {
+                                return StringUtils.startsWithIgnoreCase(deviceModel, input);
+                            }
+                        }).isEmpty();
+    }
+
+    private boolean isSupportedDeviceModel2015(final String deviceModel) {
+        return StringUtils.isNotBlank(deviceModel)
```

Comment:
Please extract the method body and parameterize the collection to be passed, so that this thing does not have to be duplicated?